tests/storage/psql_dos/test_backend.py

```diff
@@ -144,3 +144,18 @@ def mock_get_info(self, detailed=False, **kwargs):  # pylint: disable=unused-arg
     assert 'extra_value' in repository_info_out
     assert repository_info_out['value'] == 42
     assert repository_info_out['extra_value'] == 0
+
+
+@pytest.mark.usefixtures('aiida_profile_clean')
+def test_unload_profile():
+    """Test that unloading the profile closes all sqla sessions.
+
+    This is a regression test for #5506.
+    """
+    from sqlalchemy.orm.session import _sessions  # pylint: disable=import-outside-toplevel
+    manager = get_manager()
+    manager.unload_profile()
+    assert len(_sessions) == 0, str(_sessions)
+
+    # prevent following tests from crashing
+    manager.load_profile()
```

Comment:
You are implicitly relying here on the current profile for the test session to also be the default one. To be sure, would be better to get the current profile before unloading it and then explicitly loading it again at the end.

Reply:
cheers, done!

Reply:
cheers, done!